guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -5,23 +5,12 @@ This initial configuration procedure creates an organization, location, user nam
 After the initial configuration, you can create additional organizations and locations if required.
 The initial configuration also installs PostgreSQL databases on the same server.
 
-ifdef::satellite[]
 The installation process can take tens of minutes to complete.
-If you are connecting remotely to the system, use a utility such as `screen` or `tmux` that allows suspending and reattaching a communication session so that you can check the installation progress in case you become disconnected from the remote system.
+If you are connecting remotely to the system, use a utility such as `tmux` or `screen` that allows suspending and reattaching a communication session so that you can check the installation progress in case you become disconnected from the remote system.
+ifdef::satellite[]
 For more information, see https://access.redhat.com/articles/5247[How to use the screen command] or alternately the `screen` manual page.
```

Comment:
If we only keep this in master I think we should drop this line. Master should be EL8+ and screen is gone.

Reply:
And if we merge this as is, and then do a follow up with the screen removal, we can pick the non-el8-only changes cleanly ;)

Reply:
(and there *is* screen in EPEL8 which people can decide to install)